hpx/runtime/parcelset/parcelhandler.hpp

```diff
@@ -230,7 +230,7 @@ namespace hpx { namespace parcelset
         ///                 where \a err is the status code of the operation and
         ///                       \a size is the number of successfully
         ///                              transferred bytes.
-        void put_parcels(std::vector<parcel> p, std::vector<write_handler_type> f);
+//         void put_parcels(std::vector<parcel> p, std::vector<write_handler_type> f);
```

Comment:
Why so much commented out code? is this to be removed or keeping it for future use?